saleor/graphql/discount/tests/mutations/test_voucher_update.py

```diff
@@ -70,6 +66,30 @@ def test_update_voucher(staff_api_client, voucher, permission_manage_discounts):
     assert code_instance.code == new_code
 
 
+def test_update_voucher_without_code(
```

Comment:
Could we also add similar test to `voucher_create`?

Reply:
I tried to create such a test but I had problems because the logic indicates that the code field should be an empty string and we already have a test called `test_create_voucher_with_empty_code`